### PR TITLE
Improve player page

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
This uses the Plyr media player. Please see the first commit message for why I chose/tried that one. 

You can try that player here: [test deployment](https://improveplayerpage-aw1wcm92zs1wbgf5zxitcgfnzq.tobira.opencast.org/)

### CODE REVIEW

**Review by JulianKniephoff — COMMENTED** [review]
Of course this does not have to be perfect, yet, but just for the record: When I zoom in enough, the player basically disappears. :thinking:

**Comment by LukasKalbertodt** [comment]
> Of course this does not have to be perfect, yet, but just for the record: When I zoom in enough, the player basically disappears.

You mean it disappears in the vertical dimension? I.e. it's height shrinks to very low? That should be solvable by a `min-height` I guess.

And for the record: somehow this test deployment uses a buggy version of plyr that has the volume slider overlap other stuff in Firefox. Will have to investigate.

File: frontend/webpack.config.js [hunk 1]
```diff
@@ -41,6 +42,9 @@ module.exports = (_env, argv) => ({
                     icon: true,
                 },
             }],
+        }, {
+            test: /\.css$/u,
+            use: ["css-loader"],
```
Comment (JulianKniephoff) [inline frontend/webpack.config.js:47]:
```suggestion
            loader: "css-loader",
```

is a shortcut for this when there is only one loader and it doesn't need any options. We already use that, so we should be consistent here.

File: frontend/src/global.d.ts [hunk 2]
```diff
@@ -7,3 +7,8 @@ declare module "*.svg" {
     const Component: React.FC<React.SVGProps<SVGSVGElement>>;
     export default Component;
 }
+
+declare module "*.css" {
+    const value: any;
+    export default value;
+}
```
Comment (JulianKniephoff) [inline frontend/src/global.d.ts:14]: Do we want to look into alternatives to `css-loader` that would give us proper types? :thinking:

File: frontend/src/page/Video.tsx [hunk 3]
```diff
@@ -30,8 +36,74 @@ export const VideoPage: React.FC<Props> = ({ id }) => {
...
+                // We want to make sure that there is at least a bit of the content
+                // below the video player visible.
+                "& video": { maxHeight: `calc(100vh - ${HEADER_HEIGHT}px - 80px)` },
+                "& .plyr:fullscreen video": { maxHeight: "initial" },
+            },
+        }}>
+            <Plyr source={source} options={options} />
+        </div>
+    </>;
+};
```
Comment (JulianKniephoff) [inline frontend/src/page/Video.tsx:109]: If we already anticipate this being used in a video content block as well, should this just go in its own module? This module is specifically about the player **page**.

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Okay, looks good so far. Apart from the remark above, there are only a few nitpicks and/or food for thought.

**Comment by JulianKniephoff** [comment]
> You mean it disappears in the vertical dimension? I.e. it's height shrinks to very low?

Exactly.

File: frontend/src/global.d.ts (hunk 2 above)
Comment (LukasKalbertodt) [reply frontend/src/global.d.ts:14]: Ehm... do you want to? I don't see the benefit right now.

**Comment by LukasKalbertodt** [comment]
Everything (except the thing I commented) should be fixed now.

File: frontend/src/global.d.ts (hunk 2 above)
Comment (JulianKniephoff) [reply frontend/src/global.d.ts:14]: Yes, I want to, and I did: #180.

The benefit, as always with proper types, is that we find bugs. :wink:

**Review by JulianKniephoff — APPROVED** [review]
OK, cool, everything looks good now! :smile:

**Pull request merged**